### DISCUSSION TITLE
Query on Ente Auth Description and Future Monetization Plans regarding that

### DISCUSSION THREAD

**Discussion by Greatz08** [ente-io/ente#1265] [General]
https://f-droid.org/en/packages/com.beemdevelopment.aegis/   Aegis is open source authenticator app for android and i have been using it for many months and its pretty great solution and is been in market for many years so i was wondering why in main page of this project description this line was written - " Two years ago, while building Ente Photos, we realized that there was no open source end-to-end encrypted authenticator app. We already had the building blocks, so we built one.Ente Auth is currently free. If in the future we convert this to a paid service, existing users will be grandfathered in.
" .
Aegis exist plus maybe some other similar solution might have exist too i didnt researched on it as aegis solved issue for me plus all similar solutions be it open source or closed are providing for free so my question is why it is mentioned that there was no open source solution that exist plus currently it is free but you specifically mentioned about this feature that it "can become paid thing" which in my opinion should never happen in open source solution because this is small thing and is been provided for free already so it makes no sense to make this small thing paid feature in future too.you(team) can still make it paid because my opinion might not matter to you(team) but it wont make any sense to make such thing paid rest your wish.
Thanks for ente tho :-))

**Comment by mnvr** [comment]
If Aegis is solving your needs, that's great! You don't need Ente Auth then. Aegis wasn't solving our needs, which is why we built our own.

>  it "can become paid thing" which in my opinion should never happen in open source solution

I'm not sure what to say to this. You should research a bit more.

I'm not really sure what your concern is though. If you're saying that we should forever offer the auth app for free, well, we'd like to, but we cannot guarantee it, which is why we state that it may become paid in the future. There is no lock in - we're saying existing users will get grandfathered in, and we provide [easy to use ways to export your data](https://help.ente.io/auth/migration-guides/export#how-to-use-the-exported-data) if someone decides to stop using the auth app

> Thanks for ente tho :-))

❤️ Thank you!

**Reply by Greatz08** [reply]
> If Aegis is solving your needs, that's great! You don't need Ente Auth then. Aegis wasn't solving our needs, which is why we built our own.

May i know what things were missing according to you/team in aegis which ente is providing.What all Major differences are there between ente and aegis can you explain me here so that i and others can differentiate between both

>I'm not sure what to say to this. You should research a bit more.

What should i research on when i am quoting what is actually written in main page? See my Concern is simply "NOTHING".I already have aegis to solve my purpose as i explained BUT i was unable to differentiate between ente auth and aegis and why in future too there can be a possibility to make it a paid thing because authenticator has very small and important role you cant add too many things and change its purpose but if you have something innovative to implement then it can acceptable case. It is currently in my opinion not providing any thing special. See Read Carefully my message i mentioned there that team can do whatever they want simply and no one can stop so i am not stopping you or i should say i cant stop you/team, if team decide to make any thing paid feature in future they will just that if you want to make it paid in future then in my opinion you must implement some more unique things and explain users in detail and differentiate between aegis and google authenticator and ente auth else it wont make any sense.If you make things paid without differentiating it in many ways from current free great solutions which are already satisfying max then it will only tarnish this project image and not my image for been unnecessarily greedy.I hope you understand my point rest i leave this matter to team.

**Reply by vishnukvmd** [reply]
Thanks for sharing your concerns.

We wanted a cross-platform app, with e2ee sync, that's FOSS. As of writing this comment, we are the only app that ticks these boxes.

**Reply by Greatz08** [reply]
@vishnukvmd yea one unique feature which i also found later is the "cross-platform" part. Since i was seeing this project from mobile app point of view as you know max use app authenticator apps like aegis and google authenticator for authentication purposes so i also viewed this project from that perspective so couldnt find that good difference so sorry for that but yea its pretty great to be able to access authenticator totp without mobile/app while working in web so for lazy people like me it is pretty great haha :-)) 
Keep up the good work team
